include/AudioAlsa.h

```diff
@@ -80,10 +80,10 @@ class AudioAlsa : public QThread, public AudioDevice
 	static DeviceInfoCollection getAvailableDevices();
 
 private:
-	virtual void startProcessing();
-	virtual void stopProcessing();
-	virtual void applyQualitySettings();
-	virtual void run();
+	virtual void startProcessing() override;
```

Comment:
I think the `virtual` keyword is redundant if you use `override`, since `override` implicits `virtual`. I'd vote for removing all `virtual` keywords whenever you `override`, this makes the class way cleaner and faster to read.

Reply:
Before you change all this manually, consider using something like `sed 's/virtual \(.*override;\)/\1/' -i include/*`. Let me know if you need help.